src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const results = people.filter((data) =>
```

Comment:
```suggestion
  const men = people.filter((person) =>
```
Use more meaningful naming. Your code should be understandable and descriptive